operations/_scripts/deploy/deploy.sh

```diff
@@ -37,38 +37,35 @@ if [[ "$CREATE_DOMAIN" == "true" ]]; then
   /bin/bash $GITHUB_ACTION_PATH/operations/_scripts/generate/generate_acm_tf.sh
 fi
 
-TERRAFORM_COMMAND=""
-TERRAFORM_DESTROY=""
 if [ "$STACK_DESTROY" == "true" ]; then
-  TERRAFORM_COMMAND="destroy"
-  TERRAFORM_DESTROY="true"
-  ANSIBLE_SKIP_DEPLOY="true"
+  BITOPS_TERRAFORM_COMMAND="destroy"
+  BITOPS_ANSIBLE_SKIP_DEPLOY="true"
 fi
 
 if [[ "$GHA_TESTING" == "true" ]]; then
   echo "Quitting before BitOps invoke"
   exit 1
 fi
 
+# Bypass all the 'BITOPS_' ENV vars to docker
+DOCKER_EXTRA_ARGS=""
```

Comment:
The one place I thought "ahh I finally caught him" 
https://unix.stackexchange.com/questions/280430/are-the-null-string-and-the-same-string

Turns out `=""` and `=` initialize to the same thing in bash :finger_snap: